webapp/src/pages/EditAvatarPage/EditAvatarPage.css

```diff
@@ -0,0 +1,22 @@
+.edit-avatar-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  gap: 3rem var(--column-gap);
+  grid-template-columns: 1fr;
```

Comment:
From what I've read, when you have only one row or one column is better to use flex-box instead of grid. But of course this is well done.

Reply:
please, let's discuss the style in #155 .